### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by t20100**
This PR updates the README to:
- Update description of "Spawn through URL" params
- Update advanced tab screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,9 +134,8 @@ For a minimalistic working demo, check the
   where almost all Slurm job settings can be set. Some partitions can be hidden
   from the _Simple_ tab with setting `simple` to `False`.
 - `jupyter_environments`: Mapping of identifer name to information about Python
-  environment used to run Jupyter on the Slurm nodes. Either `path` or
-  `modules` (or both) should be defined. This information is a mapping
-  containing:
+  environment used to run Jupyter on the Slurm nodes. Either `path` or `modules`
+  (or both) should be defined. This information is a mapping containing:
```
Comment (t20100) [inline README.md:138]: No change but formatting tool

File: README.md [hunk 2]
```diff
@@ -213,13 +212,24 @@ The following optional query arguments are available:
...
+    Jupyter
+  - `environment_modules`: Space-separated list of
+    [environment module](https://modules.sourceforge.net/) names to load before
+    starting Jupyter
   - `root_dir`: The path of the "root" folder browsable from Jupyter(Lab)
     (user's home directory if not provided)
 
+  Note: To use a Jupyter environment defined in the configuration, only provide
+  its `environment_id`. To use a custom Jupyter environment, provide the
+  corresponding `environment_path` and/or `environment_modules`.
```
Comment (t20100) [inline README.md:231]: I hope this is clear enough. It starts to be complicated with having different options...

File: README.md [hunk 3]
```diff
@@ -213,13 +212,24 @@ The following optional query arguments are available:
...
+  - `environment_path`: Path to the Python environment bin/ used to start
+    Jupyter
+  - `environment_modules`: Space-separated list of
+    [environment module](https://modules.sourceforge.net/) names to load before
+    starting Jupyter
   - `root_dir`: The path of the "root" folder browsable from Jupyter(Lab)
     (user's home directory if not provided)
 
+  Note: To use a Jupyter environment defined in the configuration, only provide
+  its `environment_id`. To use a custom Jupyter environment, provide the
```
Comment (loichuder) [inline README.md:230]:
```suggestion
  its `environment_id`. To use a custom Jupyter environment, provide in addition the
``` 
?

File: README.md (hunk 2 above)
Comment (loichuder) [reply README.md:231]: You could give examples URL to ease the understanding

**Review by loichuder — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (t20100) [reply README.md:230]: You don't have to provide the `environment_id` for custom envs, it's more "instead" than "in addition".
There's an environment name/id for custom env in the spawn web page to display a meaningful name but it's not needed in the server

File: README.md (hunk 2 above)
Comment (t20100) [reply README.md]: Yes, good idea, done!

**Pull request merged**